manifests/init.pp

```diff
@@ -50,6 +50,8 @@
 
   $proxy_server           = undef,
   $proxy_type             = undef,
+  $proxy_host             = undef,
+  $proxy_port             = undef,
 
```

Comment:
Can we take just one format of proxy settings and combine or split them for the other case...